django-website/wheelie/client/scss/core/_reset.scss

```diff
@@ -1,5 +1,5 @@
 // Reset frameworks
-@import "../../../vendors/normalize-css/normalize";
+@import "../../../node_modules/normalize.css/normalize.css";
```

Comment:
Be sure to import the version installed via npm (or Bower if we're still using it).